maps/nerva/Loadout/loadout_accessories.dm

```diff
@@ -44,4 +44,9 @@
 /datum/gear/tactical/tacticool
 	allowed_roles = null
 
-#undef ARMED_ROLES
\ No newline at end of file
+/datum/gear/survival/survivalkit
```

Comment:
this is where your travis error is coming from. Just make it /datum/gear/survivalkit, or place it under an existing category, unless you want to fuck with travis.